source/views/sis/course-search/index.js

```diff
@@ -14,13 +14,13 @@ import {connect} from 'react-redux'
 import groupBy from 'lodash/groupBy'
 import sortBy from 'lodash/sortBy'
 import toPairs from 'lodash/toPairs'
-import CourseSearchTableView from '../components/results'
+import {CourseSearchTableView} from '../components/results'
 
 type ReactProps = TopLevelViewPropsType
 
 type ReduxStateProps = {
-	allCourses: Array<CourseType>,
-	courseDataState: string,
+	allCourses: ?Array<CourseType>,
```

Comment:
Because you always return something for these in `mapState` down below, these shouldn't need to be optional here.

Reply:
Did this because of a flow error as well...I guess state.sis.allCourses could be null in which case it would be assigned a null value (I could make it so state.sis.allCourses is never null in flux/parts/sis.js but I wasn't sure so I was basing it off the state variables in balances.js)

Reply:
Balances is special, because `0` is a valid balance; you should feel free to default to an empty array of courses, since that's how you'd represent no courses being available.